com.unity.render-pipelines.high-definition/Editor/Lighting/HDLightUI.cs

```diff
@@ -767,19 +767,32 @@ static void DrawEmissionContent(SerializedHDLight serialized, Editor owner)
             {
                 serialized.settings.DrawCookie();
 
-                // When directional light use a cookie, it can control the size
-                if (serialized.settings.cookie != null && lightType == HDLightType.Directional)
+                if (serialized.settings.cookie is Texture cookie && cookie != null)
```

Comment:
So if no texture is set, the coockie size field disappear?

Reply:
Yes, see this new UI: https://github.cds.internal.unity3d.com/unity/unity/pull/4021